go/vt/vttablet/onlineddl/vrepl/columns.go

```diff
@@ -0,0 +1,192 @@
+/*
+   Copyright 2016 GitHub Inc.
+	 See https://github.com/github/gh-ost/blob/master/LICENSE
+*/
+
```

Comment:
Has this been copied from `gh-ost` with no changes?

Reply:
Right! It was copied from `gh-ost` and there have been changes. I'll append the Vitess copyright.